discord/bot.py

```diff
@@ -532,14 +532,47 @@ def command(self, **kwargs):
         """
         return self.application_command(**kwargs)
 
-    def command_group(
-        self, name: str, description: str, guild_ids=None
+    def create_group(
+        self, name: str, description: str, guild_ids: Optional[List[int]] = None
     ) -> SlashCommandGroup:
-        # TODO: Write documentation for this. I'm not familiar enough with what this function does to do it myself.
+        """A shortcut method that creates a slash command group with no subcommands and adds it to the internal
+        command list via :meth:`~.ApplicationCommandMixin.add_application_command`.
+
+        .. versionadded:: 2.0
+
+        Returns
+        --------
+        SlashCommandGroup
+            The slash command group that was created.
+        """
```

Comment:
Wrote this on mobile so it might not be perfect. We should probably change it to accept any iterable of guild ids (if it doesn't already) and then change the type hint from `List` to `Iterable`. ```suggestion
        """A shortcut method that creates a slash command group with no subcommands and adds it to the internal
        command list via :meth:`~.ApplicationCommandMixin.add_application_command`. Previously named ``group``, but was changed to support the new functionality of :meth:`.group`.

        .. versionadded:: 2.0

        Parameters
        ----------
        name: :class:`str`
            The group name
        description: :class:`str`
            The group description
        guild_ids: Optional[List[int]]
            A list of the IDs of each guild this command should be added to, making it a guild command.

        Returns
        --------
        :class:`SlashCommandGroup`
            The slash command group that was created.
        """
```

Reply:
Yep, good idea!

Reply:
Added the parameters to the docstring, but I couldn't understand why `guild_ids` should accept all iterables. It's shown as `Optional[List[int]]` everywhere in the code.

Reply:
I guess I sorta meant it might be wise to allow guild_ids to accept all iterables everywhere.